jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -471,44 +471,21 @@ function assign_server_location(){
 #   value: "${KIE_SERVER_PORT}"
 #
 function configure_server_location() {
-    # KIE_SERVER_LOCATION matches our env-to-property naming convention,
-    # but KIE_SERVER_URL kept here for backwards compatibility
-    local location="${KIE_SERVER_LOCATION:-$KIE_SERVER_URL}"
```

Comment:
we should not remove it for backward compatibility;

Reply:
ok